check_missing_installations.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Script to check for missing installations in EESSI pilot software stack (version 2021.12)
+
+TOPDIR=$(dirname $(realpath $0))
+
+TMPDIR=$(mktemp -d)
```

Comment:
Just a minor thing, how about using a different name for `TMPDIR` ... if it would be exported it may lead to undesired side effects. Not sure if `EB_TMPDIR` would be fitting here.

Reply:
Well, that's also true for `TOPDIR` (even though the value would be the same).

This script is run in a subshell (due to the `#!/bin/bash` on top), so as long as we don't use `source check_missing_installations.sh` (and there's no reason to do so), this is fine imho.

One option could be this, to just reuse the same tmpdir if one is already set:

```bash
TMPDIR=${TMPDIR:-$(mktemp -d)}
```